### DISCUSSION TITLE
Live Audio Behavior

### DISCUSSION THREAD

**Discussion by S-Tuthill** [mcguirepr89/BirdNET-Pi#711] [General]
I like to use the Live Audio feature to listen to what the mic is hearing. It works fine. My confusion comes when I stop it for a while and then want to restart it. The on-screen icon is clearly a Pause button. And that is the behavior I see. If I Pause/Stop the audio and start it up again, I am hearing audio pick up where I paused it. What I really want to hear though is the real time Live Audio. What the mic is hearing right now. Not the audio from back when I Paused it. The only way I have found to make this happen is to go into Tools | System Controls, Reboot the pi, and start Live Audio again. Am I missing something here? It is totally plausible that there is User Error involved here. But, again, to me Live Audio seems to be a toggle on / off sort of thing. Not a Pause / Resume sort of thing.

Scott

**Comment by ehpersonal38** [comment]
Hi,

This is just a quirk in how browsers work, the audio playback element usually assumes it's playing from a static source, e.g. an .mp3 file or something, so the pause button is appropriate 90% of the time, I guess livestreams are the other 10% of the time that the developers of browsers didn't offer much support for.

Anyways, I recommend just muting the audio instead of pausing it to get the desired behavior you describe. You can do this by right clicking the tab on the top of your browser, and selecting "Mute Tab."

**Reply by S-Tuthill** [reply]
Ahh! I should have just clicked around some more.  I had a feeling there was some user error involved. The little speaker icon to the right of the clock/timer display is a Mute/Unmute button - which does exactly what I want. In addition to the approach of right clicking the tab on the top of the browser and selecting "Mute Tab". 

Thanks for the reply.
